src/constructs/kinesis/kinesis-stream.ts

```diff
@@ -0,0 +1,15 @@
+import type { CfnStream, StreamProps } from "@aws-cdk/aws-kinesis";
+import { Stream } from "@aws-cdk/aws-kinesis";
+import type { GuStack } from "../core";
+
+export interface GuKinesisStreamProps extends StreamProps {
```

Comment:
I'm not that knowledgeable about Kinesis... I wonder if we are happy with the [CFN default property values](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-kinesis-stream.html#aws-resource-kinesis-stream-properties)?

Reply:
> I wonder if we can abstract these settings at the pattern level?

I like this idea; it could be a nice future improvement for this pattern.